src/utils/quote_server_client.ts

```diff
@@ -0,0 +1,56 @@
+import { SchemaValidator } from '@0x/json-schemas';
+import { schemas, SubmitRequest } from '@0x/quote-server';
+import { Fee } from '@0x/quote-server/lib/src/types';
+import { BigNumber } from '@0x/utils';
+import { AxiosInstance } from 'axios';
+import { Summary } from 'prom-client';
+
+import { RFQT_REQUEST_MAX_RESPONSE_MS } from '../config';
+import { logger } from '../logger';
+
+const MARKET_MAKER_LAST_LOOK_LATENCY = new Summary({
+    name: 'market_maker_last_look_latency',
+    help: 'Latency for Last Look request to Market Makers',
+    labelNames: ['makerUri'],
+});
+
+export class QuoteServerClient {
+    private readonly _schemaValidator: SchemaValidator;
+
+    private static _areFeesEqual(fee1: Fee, fee2: Fee): boolean {
+        return fee1.amount.eq(fee2.amount) && fee1.token === fee2.token && fee1.type === fee2.type;
+    }
+
+    constructor(private readonly _axiosInstance: AxiosInstance) {
+        this._schemaValidator = new SchemaValidator();
```

Comment:
Why not initialize `schemaValidator` at a module level? (like you did for `MARKET_MAKER_LAST_LOOK_LATENCY`)? just a suggestion